baseapp-cloudflare-stream-field/pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[build-system]
```

Comment:
```
[tool.black]
...
[tool.isort]
...
```
sections are already in the root dir `baseapp-backend/pyproject.toml` and CI uses that one, so maybe just leave `[build-system]` section?